apps/src/applab/applab.js

```diff
@@ -541,6 +542,8 @@ Applab.init = function (config) {
 
   Applab.handleVersionHistory = studioApp().getVersionHistoryHandler(config);
 
+  var teacherViewStudentWork = getStore().getState().viewAs === ViewType.Teacher && config.readonlyWorkspace;
```

Comment:
Could be `const`.

This doesn't strike me as the most intuitive name, but I'm having a tough time coming up with something better. `teacherViewOfStudentWork`? `viewingAsTeacherViewingStudentWork`? `designModeIsUnavailable`?

You also might be able to base this entirely on `config.readonlyWorkspace` since the root cause of the issue is that we don't have a read-only design mode right now - and I suppose we wouldn't mind teachers seeing design mode if it was properly read-only.